cdk/visit/__init__.py

```diff
@@ -113,13 +116,14 @@ def log_visit_lambda(self, table_name: str):
             code=aws_lambda.Code.from_asset('visit/lambda_code/log_visit'),
             environment={
                 'TABLE_NAME': table_name,
+                'API_ENDPOINT': api_endpoint,
```

Comment:
technically this is the website domain name, not the API endpoint, right?

Reply:
Yeah sorry, I can see how that variable name could be confusing, I'll go ahead and name it something more appropriate.